fmriprep/config.py

```diff
@@ -414,6 +414,9 @@ class workflow(_Config):
     (positive = exact, negative = maximum)."""
     bold2t1w_dof = None
     """Degrees of freedom of the BOLD-to-T1w registration steps."""
+    bold2t1w_init = 'register'
```

Comment:
I *think* this is supposed to be `None` by default, but I'm not sure why. We have some `False` and `True` defaults. @oesteban Perhaps you can clarify?

```suggestion
    bold2t1w_init = None
```

Reply:
I think the vision was for these settings to be used as the default values for their parser's counterparts. Since the parser currently has `'register'` for default, there is no difference in using either value here.

However `'register'` is more informative to predict the parser's default.

Reply:
I guess we'll need to set some time to make the parser read the config module and generate the command line arguments (and help messages) from it.

Reply:
Okay. Let's leave this as is for now.